sources/install/package_base.sh

```diff
@@ -250,6 +250,12 @@ function post_install() {
     rm -rfv /tmp/*
     rm -rfv /var/lib/apt/lists/*
     rm -rfv /root/sources
+    colorecho "Stop listening processes"
+    LISTENING_PROCESSES=$(ss -lnpt | awk -F"," 'NR>1 {split($2,a,"="); print a[2]}')
```

Comment:
Let's print `LISTENING_PROCESSES` here and in the test command, so that if the test command fails we can find out what happened.

Reply:
Done